sound/soc/sof/amd/acp-dsp-offset.h

```diff
@@ -78,6 +78,9 @@
 #define ACP5X_AXI2DAGB_SEM_0			0x1884
 #define ACP6X_AXI2DAGB_SEM_0			0x1874
 
+#define ACP_SW0_I2S_ERROR_REASON		0x18B4
+#define ACP_SW1_I2S_ERROR_REASON		0x1A50
```

Comment:
what is the relationship between SoundWire and I2S? this looks odd.

Reply:
These are common ACP registers for reporting error reason for I2S interface and SDW interface i.e In these registers few register fields will refer to I2S errors and few register fields refer to SDW errors.

Reply:
add a comment then